src/ripple/core/impl/LoadMonitor.cpp

```diff
@@ -104,70 +104,33 @@ void LoadMonitor::update ()
     while (mLastUpdate < now);
 }
 
-void LoadMonitor::addCount ()
-{
-    ScopedLockType sl (mLock);
-
-    update ();
-    ++mCounts;
-}
-
-void LoadMonitor::addLatency (int latency)
-{
-    // VFALCO NOTE Why does 1 become 0?
-    if (latency == 1)
-        latency = 0;
-
-    ScopedLockType sl (mLock);
-
-    update ();
-
-    ++mLatencyEvents;
-    mLatencyMSAvg += latency;
-    mLatencyMSPeak += latency;
-
-    // Units are quarters of a millisecond
-    int const latencyPeak = mLatencyEvents * latency * 4;
-
-    if (mLatencyMSPeak < latencyPeak)
-        mLatencyMSPeak = latencyPeak;
-}
-
-std::string LoadMonitor::printElapsed (double seconds)
-{
-    std::stringstream ss;
-    ss << (std::size_t (seconds * 1000 + 0.5)) << " ms";
-    return ss.str();
-}
-
 void LoadMonitor::addLoadSample (LoadEvent const& sample)
 {
-    std::string const& name (sample.name());
-    beast::RelativeTime const latency (sample.getSecondsTotal());
+    using namespace std::chrono_literals;
 
-    if (latency.inSeconds() > 0.5)
+    auto latency = sample.getSecondsTotal();
+
+    if (latency > 500ms)
     {
-        auto mj = latency.inSeconds() > 1.0 ? j_.warn() : j_.info();
-        JLOG (mj)
-            << "Job: " << name << " ExecutionTime: " << printElapsed (sample.getSecondsRunning()) <<
-            " WaitingTime: " << printElapsed (sample.getSecondsWaiting());
+        auto mj = (latency > 1s) ? j_.warn() : j_.info();
+        JLOG (mj) << "Job: " << sample.name() <<
+            " ExecutionTime: " << sample.getSecondsRunning().count() << "ms" <<
+            " WaitingTime: " << sample.getSecondsWaiting().count() << "ms";
     }
 
     // VFALCO NOTE Why does 1 become 0?
-    std::size_t latencyMilliseconds (latency.inMilliseconds());
-    if (latencyMilliseconds == 1)
-        latencyMilliseconds = 0;
+    latency = std::min(latency, 1ms);
```

Comment:
I've made changes which I believe make this code unnecessary: `LoadEvent` now tracks the time using the full resolution available from `std::chrono::steady_clock`. We then convert that time using `ceil<std::chrono::duration>` to round results up to the nearest millisecond.